### PR TITLE
Milestone#2: Create custom methods

### PR DESCRIPTION
**Pull request by carlylechia**
### In this branch, I added the following data processing methods to their respective models:

- [x] Add the `last_three_posts` method into the **users model**

- [x] Add the `last_five_comments` method into the **posts model**

The `comments_count` for the **posts model**, the `posts_count` for the **users model**, and the `likes_count` for the **posts model** are ***automagically*** updated. So I didn't add them.

Thank you for taking a look 😄!

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -2,4 +2,8 @@ class Post < ApplicationRecord
   belongs_to :user, counter_cache: true
   has_many :comments
   has_many :likes
+
+  def last_five_comments
+    comments.limit(5).order(created_at: :desc)
+  end
```
Comment (DeliceLydia) [inline app/models/post.rb:10]: - [x] Please work on updating the posts counter method you can do something like:
```ruby
def update_post_counter
  author.increment!(:posts_counter)
end
```

Comment (DeliceLydia) [reply app/models/post.rb:10]: ### Optional
- [x] I suggest that you use [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the `#update_post_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private

Comment (DeliceLydia) [reply app/models/post.rb:10]: - [x] In the `comment` model you are supposed to add a method that updates the comments counter for a post you can do:
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```

Comment (DeliceLydia) [reply app/models/post.rb:10]: ## Optional
- [x] I suggest that you use [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the `#update_comments_counter` method every time a new `comment` has been saved to the database, then make the `update_comments_counter` method private

Comment (DeliceLydia) [reply app/models/post.rb:10]: - [x] In the `like` method you are supposed to add a method that updates the likes counter for a post, you can do:
```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```

Comment (DeliceLydia) [reply app/models/post.rb:10]: ## Optional
- [x] I suggest that you use [Rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save` to call the `#update_likes_counter` method every time a new `like` has been saved to the database, then make the `update_likes_counter` method private

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @carlylechia, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there! Highlights for your project:
- Awesome README
- Followed git workflow
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/models/post.rb (hunk 1 above)
Comment (carlylechia) [reply app/models/post.rb:10]: Thank you so much @DeliceLydia, I'll implement the changes you've so modestly requested.

**Pull request merged**